src/main/java/bio/terra/cli/command/shared/options/NewCloningInstructions.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.cli.command.shared.options;
+
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import picocli.CommandLine;
+
+public class NewCloningInstructions {
+  /**
+   * Command helper class that defines the project id and dataset id when updating a BigQuery
```

Comment:
This description was copied over from another class. Please change it to describe how this mixin is used (e.g. that it's used for both bucket and dataset update). Also note that it's not intended for resource Create; we have other mixins for that.

Reply:
Looks like is mixin is for controlled resources, how about calling this `ControlledCloningInstructions.java`? I'm not sure what `New` in `NewClongingInstructions` means.

```
package bio.terra.cli.command.shared.options;

import bio.terra.workspace.model.CloningInstructionsEnum;
import picocli.CommandLine;

public class ControlledCloningInstructions {
  /**
   * Use this --cloning flag for controlled resources.
   *
   * <p>This class is meant to be used as a @CommandLine.Mixin.
   */

```